File: DesktopApp/Model/Directory.cs
using Newtonsoft.Json;

namespace ProfessorPerformanceEvaluation.Model
{
    public class Directory
    {
        [JsonProperty("idDirectory")]
        public int IdEducationalExperience { get; set; }
        [JsonProperty("idFaculty")]
        public int IdFaculty { get; set; }
        [JsonProperty("idProfessor")]
        public int IdProfessor { get; set; }
    }
}

File: MobileApp/app/src/main/java/com/example/professorperformanceevaluation/model/Directory.java
package com.example.professorperformanceevaluation.model;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Directory {

    @SerializedName("idDirectory")
    @Expose
    private int idDirectory;
    @SerializedName("idFaculty")
    @Expose
    private int idFaculty;
    @SerializedName("idProfessor")
    @Expose
    private int idProfessor;

    public Directory() {
    }

    public int getIdDirectory() {
        return idDirectory;
    }

    public void setIdDirectory(int idDirectory) {
        this.idDirectory = idDirectory;
    }

    public int getIdFaculty() {
        return idFaculty;
    }

    public void setIdFaculty(int idFaculty) {
        this.idFaculty = idFaculty;
    }

    public int getIdProfessor() {
        return idProfessor;
    }

    public void setIdProfessor(int idProfessor) {
        this.idProfessor = idProfessor;
    }

}